Removed statement around unregistering application

suneetnangia opened the pull request:
"Unregistering unused application types releases storage space used by the image store" this is not true in my opinion as you can still register the application type after unregistering it which means the storage space is not released.

azurecla commented:
Hi __@ggailey777__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (glenga). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

azurecla commented:
Hi __@ggailey777__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (glenga). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

suneetnangia committed:
Removed statement around unregistering application

"Unregistering unused application types releases storage space used by the image store" this is not true in my opinion as you can still register the application type after unregistering it which means the storage space is not released.
